change/@internal-react-composites-3e8203bf-9f73-4300-8ae4-34b282a1e57a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Should be `prerelease`.

`minor` makes change to the stable build.

Correct me if I am wrong @JamesBurnside @prprabhu-ms